app-frontend/src/app/components/projects/aoiDrawToolbar/index.html

```diff
@@ -0,0 +1,94 @@
+<div class="navbar light-navbar"
+     ng-style="$ctrl.eleStyle">
+  <div class="navbar-section aoi-draw-toolbar-left"
+       ng-style="$ctrl.leftStyle">
+    <button class="btn btn-small btn-transparent"
+            ng-click="$ctrl.onDrawAoi()"
+            ng-disabled="$ctrl.isDrawingAoi"
+            ng-if="!$ctrl.layerAoiGeom">
+      <i class="icon-plus"></i> Draw AOI
+    </button>
+    <button class="btn btn-small btn-transparent"
+            ng-if="$ctrl.layerAoiGeom"
+            ng-disabled="$ctrl.isEditingAoi"
+            uib-dropdown
+            uib-dropdown-toggle>
+      <i class="icon-pencil"></i>Edit AOI<i class="icon-caret-down"></i>
+      <ul class="dropdown-menu dropdown-menu-light drop-left aoi-draw-dropdown"
+          role="menu"
+          uib-dropdown-menu >
+        <li role="menuitem">
+          <a class="aoi-draw-toolbar-menu"
+             ng-click="$ctrl.onClickEditAoi()">
+             Edit
+          </a>
+        </li>
+        <li role="menuitem">
+          <a class="aoi-draw-toolbar-menu"
+             ng-click="$ctrl.onClickDeleteAoi()">
+             Delete
+          </a>
+        </li>
+      </ul>
+    </button>
+    <div style="flex: 1;"></div>
+    <div class="btn btn-small btn-transparent"
```

Comment:
After seeing this, I think this should be a `<select>` instead of a dropdown. This would have the advantage of displaying the current selection. This also means we could remove the list header that says the type of thing being filtered to in the `.../aoi/index.html` file.

Reply:
@designmatty your input here would be good 😃